eng/Publishing.props

```diff
@@ -8,7 +8,6 @@
     <Product>Sdk</Product>
     <BlobStoragePartialRelativePath>$(Product)</BlobStoragePartialRelativePath>
     <BlobStoragePartialRelativePath Condition="'$(IsNotOrchestratedPublish)' == 'false'">assets/$(Product)</BlobStoragePartialRelativePath>
-    <PublishToBlobFeedFlatContainer>true</PublishToBlobFeedFlatContainer>
```

Comment:
Removing this line might make the publishing of the zips/tarballs/native installers etc break and be pushed to the root of the blob storage?

Did you run a build locally pointing to a fake blob storage to validate everything published correct?

Reply:
The removal of the line only affects the arcade based publishing. Since the parameter is manually set to true in both https://github.com/dotnet/core-sdk/blob/593ca32f658828e0a16a7d7100cc450e79c07911/eng/Publishing.props#L87 and https://github.com/dotnet/core-sdk/blob/593ca32f658828e0a16a7d7100cc450e79c07911/eng/Publishing.props#L99, it's not affecting what gets published to the cli and clichecksums storage accounts.

I did run some tests and the custom publishing didn't put things in the root. I already deleted those containers, but I'll spin up another build that publishes to a test storage account and we can take a look at it to see if everything looks right in the feeds.